syntax = "proto3";

message FindGatewayRequest {

}